app/Http/Controllers/ControllerDash.php

```diff
@@ -509,26 +510,23 @@ public function searchAirportResult(Request $request) {
         }
 
         $client = new Client(['http_errors' => false]);
-        $res = $client->request('GET', 'https://api.aviationapi.com/v1/charts?apt='.$apt_r);
+        $res = $client->request('GET', 'https://api-v2.aviationapi.com/v2/charts?airport='.$apt_r);
         $status = $res->getStatusCode();
         if ($status == 404) {
             $charts = null;
         } elseif (json_decode($res->getBody()) != '[]') {
-            $charts = collect(json_decode($res->getBody())->$apt_r);
-            $min = $charts->where('chart_code', 'MIN');
-            $hot = $charts->where('chart_code', 'HOT');
-            $lah = $charts->where('chart_code', 'LAH');
-            $apd = $charts->where('chart_code', 'APD');
-            $iap = $charts->where('chart_code', 'IAP');
-            $dp = $charts->where('chart_code', 'DP');
-            $star = $charts->where('chart_code', 'STAR');
-            $cvfp = $charts->where('chart_code', 'CVFP');
+            $charts = collect(json_decode($res->getBody())->charts);
+            $general = $charts['general'];
```

Comment:
I like how you combine all of the misc charts into the 'general' category in v2 - nicely done.